dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/event/CopyActionletEvent.java

```diff
@@ -0,0 +1,24 @@
+package com.dotmarketing.portlets.workflows.actionlet.event;
+
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+
+import java.io.Serializable;
+
+public class CopyActionletEvent implements Serializable {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)